scripts/export-dos-specialists.py

```diff
@@ -72,10 +85,13 @@ def get_specialist_roles(section):
 
     pool = ThreadPool(3)
 
+    logger.info(f"Finding Digital Specialists suppliers for {FRAMEWORK_SLUG}")
     suppliers = find_all_specialists(client, map_impl=pool.imap)
 
+    logger.info(f"Building CSV for {len(suppliers)} Digital Specialists suppliers")
     write_csv_with_make_row(
         suppliers,
         make_row(content_manifest),
-        "output/{}-specialists.csv".format(FRAMEWORK_SLUG)
+        os.path.join(OUTPUT_DIR, "digital-specialists-suppliers.csv".format(FRAMEWORK_SLUG)),
```

Comment:
There isn't a site for the format method to insert the framework slug in this string?